external/dhcpd/dhcpd_lwip.c

```diff
@@ -217,7 +229,7 @@ int dhcp_server_start(char *intf, dhcp_sta_joined dhcp_join_cb)
 	pthread_setname_np(g_dhcpd_tid, "dhcpd cb handler");
 	if (ret < 0) {
 		g_dhcpd_term = 1;
-		ndbg("[dhcpd] create iotapi handler fail(%d) errno %d\n", ret, errno);
+		ndbg("[DHCPD] create iotapi handler fail(%d) errno %d\n", ret, errno);
```

Comment:
In this case, you'd better free `data->intf` and `data`

Reply:
Thanks for the comment. Applied.